### ISSUE TITLE
Comic Easel Page Content 

### ISSUE THREAD

**Issue by ryank119** [Frumph/easel#34]
On my custom pages the text, images and comments are right on the left edge of the page unless they are centred.  I wanted to know how to format the pages by default so there is some white space between the edge and the content.

**Comment by Frumph** [comment]
Best kinda closing ;/ quick and painless.  Did you do a padding?

**Comment by ryank119** [comment]
Haha yea, I ended up doing width, padding and margins on the #comment tags.  Worked excellently.

**Comment by ryank119** [comment]
Hey Frumph, came across another problem.  Without thinking I edited my functions.php on Comic Easel rendering my site useless.  Where can I get a fresh functions.php?  I downloaded Easel and unzipped it, but I couldn't find that one in there.
Thanks!

 

Subject: Re: [easel] Comic Easel Page Content  (#34)
From: ryank_119@hotmail.com
Date: Tue, 21 Jul 2015 20:42:18 -0700
To: reply@reply.github.com

Haha yea, I ended up doing width, padding and margins on the #comment tags.  Worked excellently.



On Jul 21, 2015, at 8:22 PM, Philip M. Hofer (Frumph) <notifications@github.com> wrote:

Best kinda closing ;/ quick and painless.  Did you do a padding?

**Comment by ryank119** [comment]
I have to close this one early too, haha.  I ended up fixing it by downloading the functions.php from my FTP server and removing edits and reuploading again.  Thanks anyways! 
Loving Easel, btw.  So much functionality for me!
Ryan Klassenhttp://cityfolk.ca

 

From: ryank_119@hotmail.com
To: reply@reply.github.com
Subject: RE: [easel] Comic Easel Page Content  (#34)
Date: Thu, 30 Jul 2015 08:02:40 -0700




Hey Frumph, came across another problem.  Without thinking I edited my functions.php on Comic Easel rendering my site useless.  Where can I get a fresh functions.php?  I downloaded Easel and unzipped it, but I couldn't find that one in there.
Thanks!

 

Subject: Re: [easel] Comic Easel Page Content  (#34)
From: ryank_119@hotmail.com
Date: Tue, 21 Jul 2015 20:42:18 -0700
To: reply@reply.github.com

Haha yea, I ended up doing width, padding and margins on the #comment tags.  Worked excellently.



On Jul 21, 2015, at 8:22 PM, Philip M. Hofer (Frumph) <notifications@github.com> wrote:

Best kinda closing ;/ quick and painless.  Did you do a padding?
